### DISCUSSION TITLE
Donate/Sponsor

### DISCUSSION THREAD

**Discussion by fruktiz32** [Bodmer/TFT_eSPI#2242] [General]
Hello. Thanks for your great library I've been using for long time. Want to ask where I can donate some beer money because I can't find, please help. Thank you for your great work!

**Comment by Bodmer** [comment]
Thank you for your kind comments, they are appreciated.

If I did accept donations then I would pass them on to charities, so it is simpler if you just make a donation to a charity of your choice. Your thanks is enough for me.

**Comment by TangerineDreamer** [comment]
I would like also to pay some cofees to you cause i appreciate your library and your patience too :)  👍
